source/javascript/src/weekly.js

```diff
@@ -66,13 +66,16 @@ function populateDayColumns (weeklyItems) {
   const current = new DateConverter(Date.now())
   // Use a new instance of Date to fetch the day of the week of today
   const today = new DateConverter()
-  const todayDays = today.getDay()
+  let todayDays = today.getDay()
   weeklyItems.forEach((entry) => {
     // calculate the offset between today's day and the entry's day
     const offSet = current.getDaysFromTimeStamp(Date.now()) - current.getDaysFromTimeStamp(entry.properties.date.time)
     const currentDay = new DateConverter(Number(entry.properties.date.time))
     // apply the offset to get the index
-    const index = todayDays + 6 - offSet
```

Comment:
Depending on the order in which we merge #264 and #262, this section may create merge conflicts.
We should also have a quick meeting on Tuesday around standup to discuss what the best approach would be here  (Sunday-Saturday vs. Monday-Sunday)